File: nan21.dnet.module.md/nan21.dnet.module.md.ui.extjs/src/main/webapp/src/ds/AccountLovDs.js
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
Ext.define("net.nan21.dnet.module.md.acc.ds.model.AccountLovDs" ,{
	extend: 'Ext.data.Model',
	fields: [
		{name:"accSchemaId", type:"int", useNull:true},
		{name:"active", type:"boolean"},
		{name:"clientId", type:"int", useNull:true},
		{name:"code", type:"string"},
		{name:"id", type:"int", useNull:true},
		{name:"name", type:"string"}
	]});
Ext.define("net.nan21.dnet.module.md.acc.ds.model.AccountLovDsFilter" ,{
	extend: 'Ext.data.Model',
	fields: [
		{name:"accSchemaId", type:"int", useNull:true},
		{name:"accSchemaId_From",type:"int", useNull:true},
		{name:"accSchemaId_To",type:"int", useNull:true},
		{name:"active", type:"boolean", useNull:true},
		{name:"code", type:"string"},
		{name:"id", type:"int", useNull:true},
		{name:"name", type:"string"}
	]
});
Ext.define("net.nan21.dnet.module.md.acc.ds.param.AccountLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
});

File: nan21.dnet.module.md/nan21.dnet.module.md.presenter/src/main/java/net/nan21/dnet/module/md/acc/ds/model/AccountLovDs.java
/* 
 * DNet eBusiness Suite
 * Copyright: 2010-2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.md.acc.ds.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeWithCodeLov;
import net.nan21.dnet.module.md.acc.domain.entity.Account;

@Ds(entity = Account.class, jpqlWhere = " e.active = true ", sort = {@SortField(field = AccountLovDs.f_code)})
public class AccountLovDs extends AbstractTypeWithCodeLov<Account> {

	public static final String f_accSchemaId = "accSchemaId";

	@DsField(join = "left", path = "accSchema.id")
	private Long accSchemaId;

	public AccountLovDs() {
		super();
	}

	public AccountLovDs(Account e) {
		super(e);
	}

	public Long getAccSchemaId() {
		return this.accSchemaId;
	}

	public void setAccSchemaId(Long accSchemaId) {
		this.accSchemaId = accSchemaId;
	}
}
